pkg/funk/slice.go

```diff
@@ -81,6 +81,16 @@ func Contains[T any](in []T, elem T) bool {
 	return ContainsFunc(in, equalTo)
 }
 
+func ContainsAll[T any](left []T, right []T) bool {
```

Comment:
tbh I find it unintuitive that we check that left is a subset of right and not the otherway round.
Typically in the stdlib its the otherway round, you first specify the set and then the individual elements.
At least we would need to add a comment documenting if we deviate from that.

Reply:
adjusted